android-components/components/concept/awesomebar/src/main/java/mozilla/components/concept/awesomebar/AwesomeBar.kt

```diff
@@ -192,14 +192,16 @@ interface AwesomeBar {
      * @property providers The list of [SuggestionProvider]s in this group.
      * @property priority An optional priority for this group. Decides the order of this group
      * in the AwesomeBar suggestions. Group having the highest integer value will have the highest priority.
+     * By default, it should have -1 priority. If AwesomeBar.Suggestion list is not empty group.priority
+     * will be the first suggestion score from the list.
```

Comment:
To reviewers, I think we need another on this comment since I am not quite sure I am understanding it.

```suggestion
     * By default, it should have -1 priority. If [Suggestion] list is not an empty group, priority
     * will be the first suggestion score from the list.
```

Reply:
Done